src/theme_set.js

```diff
@@ -145,12 +191,23 @@ class ThemeSet {
         .reverse()
     }
 
-    return [
+    const props = [
       get(themeInstance, propPath),
       ...importedProps,
       get(this.default, propPath),
```

Comment:
I've noticed that meta value should not fallback to default theme because that affects only in theme CSS.

`#getThemeProp` is still required by inline SVG mode. We might have to split into another method like `#getMeta`.